npm/angular/README.md

```diff
@@ -1,221 +1,154 @@
-# @cypress/angular
```

Comment:
Should clean this up to be a minimal README like the other mount packages since most of this information will be in the official docs.